airbyte-integrations/connectors/destination-jdbc/src/main/resources/ssh-tunnel-spec.json

```diff
@@ -0,0 +1,167 @@
+{
+  "type": "object",
+  "title": "SSH Tunnel Method",
+  "description": "Whether to initiate an SSH tunnel before connecting to the database, and if so, which kind of authentication to use.",
+  "oneOf": [
+    {
+      "title": "No Tunnel",
+      "required": ["tunnel_method"],
+      "properties": {
+        "tunnel_method": {
+          "description": "No ssh tunnel needed to connect to database",
+          "type": "string",
+          "enum": ["NO_TUNNEL"],
```

Comment:
can we use `const: "NO_TUNNEL"` instead of `enum`/`default` like described [here](https://docs.airbyte.io/connector-development/connector-specification-reference)? `enum`/`default` shows two dropdowns to the user which is confusing. 

Separately it might be better for the UI to name all these options without undesrcores e.g `No Tunnel`, `SSH Key Auth`, `SSH Password Auth` etc.. since it's more user friendly